src/components/Image.js

```diff
@@ -109,16 +111,23 @@ export const Image = ({
         resizeMode={resizeMode}
         borderRadius={borderRadius}
       >
-        {!!message && <ImageMessage message={message} />}
-        {!!globalSettings?.showImageRights && !!sourceProp?.copyright && (
-          <ImageRights imageRights={sourceProp.copyright} />
-        )}
+        <View style={styles.contentContainerStyle}>
```

Comment:
i think that this view should only be rendered if one of its children will be presented

Reply:
I don't know exactly what you mean.

Reply:
this view is rendered always, no matter if there are children or not. this could be changed.

Reply:
I don't think there would be a problem with an empty view appearing. Do you want me to add such a control?
```
{!!message && !!button && !!globalSettings?.showImageRights && !!sourceProp?.copyright && (
          <View style={styles.contentContainerStyle}>
...
)}
```

Reply:
"if you do not see it, its fine" is not a proper reason. think about not rendering stuff we do not want to be processed and present, even if it is not visible. yes, you need a condition to render the view or not. but not with `&&` because there are cases, where just one of those should be presented, right?!

Reply:
Then I suggest such a check.


```
`const showChildren = !!message || !!button || (!!globalSettings?.showImageRights && !!sourceProp?.copyright);`
...
        {showChildren && (
          <View style={styles.contentContainerStyle}>
            {!!message && <ImageMessage message={message} />}
            {!!button && <ImageButton button={button} />}
            {!!globalSettings?.showImageRights && !!sourceProp?.copyright && (
              <ImageRights imageRights={sourceProp.copyright} />
            )}
          </View>
        )}
```

Reply:
ok. and what about adding a `const showImageRights = !!globalSettings?.showImageRights && !!sourceProp?.copyright` to avoid duplication of that condition?

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/957/commits/02549cf49e5f6e6b2ddd17cb0c79e4c48f4937b9